tools/cli/go-whisk-cli/wski18n/resources/en_US.all.json

```diff
@@ -1126,5 +1126,29 @@
   {
     "id": "An API host must be provided.",
     "translation": "An API host must be provided."
+  },
+  {
+    "id": "Invalid projection field '{{.arg}}'.",
```

Comment:
Please check with the design team regarding the use of the word `projection`.  This seems more like a `field filter` to me.